### PR TITLE
initial noble migration

### PR DESCRIPTION
**Pull request by ramonskie**

### CODE REVIEW

File: content/noble-migration.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+Cloud Foundry's upcoming stemcells will be based on Ubuntu's [Noble Numbat](https://wiki.ubuntu.com/Releases) release, which may cause compilation and deployment errors in packages built for earlier stemcells. This document provides guidance on how to address the most common errors that BOSH release authors may encounter. There are three broad categories to address:
```
Comment (benjaminguttmann-avtq) [inline content/noble-migration.md:1]: you write "three broad categories" but you list four below.

Comment (ramonskie) [reply content/noble-migration.md:1]: yeah i can better remove that part as there are probably more things coming

**Review by lnguyen — APPROVED** [review]

**Review by benjaminguttmann-avtq — APPROVED** [review]
LGTM

**Comment by rkoster** [comment]
The link does not seem to work: https://bosh.io/docs/noble-migration

**Comment by rkoster** [comment]
Found here: https://bosh.io/stemcells/bosh-aws-xen-hvm-ubuntu-noble

**Comment by benjaminguttmann-avtq** [comment]
I think the doc needs to be listed here: https://github.com/cloudfoundry/docs-bosh/blob/master/mkdocs.yml

What would be the best section to include it then I would prepare a PR for it

**Comment by ramonskie** [comment]
@benjaminguttmann-avtq you are correct. i forget that part.

**Pull request merged**